android/app/build.gradle

```diff
@@ -172,7 +172,12 @@ android {
 }
 
 dependencies {
-  implementation project(':react-native-login-with-amazon')
+   //  compile project(':react-native-vector-icons')
```

Comment:
not your fault but remove these commented lines or any other duplicate implementations (react-native link is dumb at this)